bundle/jsky.app.ot/src/main/scala/jsky/app/ot/gemini/ghost/GhostEditor.scala

```diff
@@ -382,15 +382,39 @@ final class GhostEditor extends ComponentEditor[ISPObsComponent, Ghost] {
     object targetPane extends GridBagPanel {
       var row = 0
       /**
-       * Enable fiber agitator
+       * Enable fiber agitators
        */
-      val enableFiberAgitatorCtrl: CheckboxPropertyCtrl[Ghost] = new CheckboxPropertyCtrl[Ghost](Ghost.ENABLE_FIBER_AGITATOR_PROP)
-      layout(Component.wrap(enableFiberAgitatorCtrl.getComponent)) = new Constraints() {
+      val fiberAgitatorPanel = new GridBagPanel
```

Comment:
It is wise to use `GridBagPanel`, nobody can really review the code with `GridBagPanel` 😄

Reply:
I actually kind of love the `GridBagPanel`. I guess the result of reading a 1280 page book on Swing from 2002 before more sophisticated layout managers like `MigLayout` were around. On top of that, we're both old and clunky, so we have a bond.